frontend/next.config.mjs

```diff
@@ -2,8 +2,16 @@
 const nextConfig = {
   reactStrictMode: true,
   output: 'standalone',
+
   images: {
-    domains: ['placehold.co'],
+    remotePatterns: [
```

Comment:
What uses this?

Reply:
There are files in public images folder for example logo we need this settings for Image to work. Need to update to add tdt demo which I will add after merge and if I find any other issues